examples/gallery/src/arc-layer.html

```diff
@@ -12,27 +12,19 @@
         width: 100vw;
         height: 100vh;
         margin: 0;
+        overflow: hidden;
       }
-      #tooltip:empty {
-        display: none;
-      }
-      #tooltip {
+      .deck-tooltip {
```

Comment:
Can't we get rid of these completely?

Reply:
I prefer to leave them here so people know how they can override the default styles.